### PR TITLE
add option to assign or change constructed deck URLs

### PR DESCRIPTION
**Pull request by percentcer**
Sometimes you want to update your deck from a different URL than the original import

### CODE REVIEW

**Comment by Epix37** [comment]
Nice idea.

I see two problems right now:

* "rebase" might not be very clear for the average user. Something like "update url" or "set url" is probably easier to understand.
* The menu item is currently missing from the `DECKS` menu in the menu bar.

**Comment by percentcer** [comment]
Ah, good point.  Updated to use the wording "LINK TO NEW URL" and "LINK TO URL" (if the deck does or does not already have a URL, respectively).

Also added the option to the "DECK" menu (required a slight change to the DeckPickerList_OnSelectedDeckChanged call to update the menu items accordingly)

File: Hearthstone Deck Tracker/Windows/MainWindow/MainWindow_Edit.cs [hunk 1]
```diff
@@ -317,6 +317,13 @@ internal void BtnEditDeck_Click(object sender, RoutedEventArgs e)
 			TagControlEdit.SetSelectedTags(deck.Tags);
 		}
 
+		internal async void BtnSetDeckUrl_Click(object sender, RoutedEventArgs e)
+		{
+			var selectedDeck = DeckPickerList.SelectedDecks.FirstOrDefault();
+			selectedDeck.Url = await InputDeckURL();
```
Comment (Epix37) [inline Hearthstone Deck Tracker/Windows/MainWindow/MainWindow_Edit.cs]: One more thing: selectedDeck *could* in theory be null here. I think in reality there is no way for this method to be called without a deck being selected, but `FirstOrDefault()` should be follow by a null check nonetheless.

**Comment by percentcer** [comment]
Added a deck null check and emptystring check to the BtnSetDeckUrl_Click function

**Comment by Epix37** [comment]
Perfect! :+1:

**Pull request merged**